src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -358,6 +360,20 @@ private String createIssue(ScanResults.XIssue issue, ScanRequest request) throws
         }
     }
 
+    private String getScannerTypeSeverity(ScanResults.XIssue issue, String severity, List<ScanResults.ScaDetails> scaDetails) {
+        String scannerTypeSeverity;
+        if (ScanUtils.isSAST(issue)) {
+            scannerTypeSeverity = severity;
+        } else {
+            /*
+                Format should be aligned with SAST format
+                In SCA case - HIGH -> High
+             */
+            scannerTypeSeverity = StringUtils.capitalize(Objects.requireNonNull(scaDetails).get(0).getFinding().getSeverity().name().toLowerCase());
```

Comment:
Need to check if scaDetails can't be null

Reply:
In case SCA scan ends with not results we won't reach this code section